.github/workflows/ci-go.yml

```diff
@@ -50,3 +50,26 @@ jobs:
           go get .
           gosec -exclude=G204,G304,G107 -exclude-dir=protobuf/ ./...  
         working-directory: src
+
+  go-ginkgo-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: v1.18
+        
+      - name: Setup a Kubernetes environment
+        run: ./.github/workflows/install-k3s.sh
+
+      - name: Run KubeArmor in background
+        run: |
+          make -C KubeArmor/ run &
+          bash -c 'for((i=0;i<300;i++)); do cnt=$(ps -e | grep kubearmor | wc -l); [[ $cnt -gt 0 ]] && echo "Kubearmor is running" && break; sleep 1; done'
+        timeout-minutes: 5
+
+      - name: Run ginkgo tests
+        run: ginkgo -r
```

Comment:
Do we need to install ginkgo first? Is this the reason https://github.com/Prateeknandle/discovery-engine/actions/runs/3891874311/jobs/6642631675 failing?

Reply:
resolved